### PR TITLE
Update scratch pvc name

### PR DESCRIPTION
**Pull request by dalia-frank**
##### Short description:
  When DV deployment done with populators the scratch PVC name is prime-<pvc_uid>-scratch

##### What this PR does / why we need it:
Fix scratch_pvc to work with both DV's deployed with populators and not.

### CODE REVIEW

File: ocp_resources/datavolume.py [hunk 1]
```diff
@@ -252,8 +252,16 @@ def pvc(self):
 
     @property
     def scratch_pvc(self):
+        scratch_pvc_prefix = (
+            f"prime-{self.pvc.instance.metadata.uid}"
+            if self.pvc.instance.metadata.annotations.get(
```
Comment (rnetser) [inline ocp_resources/datavolume.py:257]: `self.pvc.instance.metadata` - save and re-use

File: ocp_resources/datavolume.py [hunk 2]
```diff
@@ -252,8 +252,16 @@ def pvc(self):
...
+                "cdi.kubevirt.io/storage.usePopulator"
```
Comment (rnetser) [inline ocp_resources/datavolume.py:258]: use `CDI_KUBEVIRT_IO` ApiGroup

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by rnetser — CHANGES_REQUESTED** [review]
.

File: ocp_resources/datavolume.py (hunk 1 above)
Comment (dalia-frank) [reply ocp_resources/datavolume.py]: ack

File: ocp_resources/datavolume.py [hunk 3]
```diff
@@ -252,8 +252,17 @@ def pvc(self):
 
     @property
     def scratch_pvc(self):
+        pvc_metadata = self.pvc.instance.metadata
+        scratch_pvc_prefix = (
+            f"prime-{pvc_metadata.uid}"
+            if pvc_metadata.annotations.get(
+                f"{NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO}/storage.usePopulator"
```
Comment (myakove) [inline ocp_resources/datavolume.py:259]:
```suggestion
                f"{self.ApiGroup.CDI_KUBEVIRT_IO}/storage.usePopulator"
```

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**